contracts/xsgd_contract.scilla

```diff
@@ -370,11 +375,10 @@ transition lawEnforcementWipingBurn(address : ByStr20, initiator : ByStr20)
       | Some b =>
         balances[address] := zero;
         currentSupply <- totalSupply;
-        new_supply = builtin sub currentSupply bal;
+        new_supply = builtin sub currentSupply b;
```

Comment:
```suggestion
        new_supply = builtin sub currentSupply bal;
```

Reply:
This is fixed in https://github.com/Xfers/XSGD-scilla/pull/13 along with line 380 following